README.md

```diff
@@ -62,30 +69,43 @@ bundle install --path vendor/bundle
 npm i babel-minify
 ```
 
-## Running locally :running:
+## Building :running:
 
-Ubuntu/WSL/MacOS:
+Docker (Windows/Linux/MacOS):
+
+```BASH
+docker run -p 4000:4000 -v $(pwd):/twofactorauth
```

Comment:
Something is wrong or missing here. When I try to run the docker comands, I get this:
```
> docker pull 2factorauth/twofactorauth:4.0
4.0: Pulling from 2factorauth/twofactorauth
Digest: sha256:b47e1773f292b6c76f99525feaff27cd2a5013812478846e84c3da03d249d270
Status: Image is up to date for 2factorauth/twofactorauth:4.0
docker.io/2factorauth/twofactorauth:4.0

> docker run -p 4000:4000 -v $(pwd):/twofactorauth
"docker run" requires at least 1 argument.
See 'docker run --help'.

Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Run a command in a new container
```
(Docker version 20.10.2)

Reply:
I believe that the command is meant to be
`docker run -p 4000:4000 -v $(pwd):/twofactorauth 2factorauth/twofactorauth:4.0`

but even after running that, there still seems to be an issue:
```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "./_deployment/entrypoint.sh": stat ./_deployment/entrypoint.sh: no such file or directory: unknown.
ERRO[0000] error waiting for container: context canceled
```
From what I can tell, the [image on Docker Hub](https://hub.docker.com/layers/2factorauth/twofactorauth/4.0/images/sha256-b47e1773f292b6c76f99525feaff27cd2a5013812478846e84c3da03d249d270?context=explore) is outdated and still refers to `_deployment/entrypoint.sh` which is causing an error.